topics/single-cell/tutorials/alevin-commandline/preamble.md

```diff
@@ -23,6 +23,12 @@ As a recap, we will go from raw FASTQ files to a cell x gene data matrix in AnnD
 
 > {% snippet faqs/galaxy/interactive_tools_jupyter_launch.md %}
 
+> <comment-title>JupyterLab Notebook version</comment-title>
+> It is important that you check the version of JupyterLab Notebook tool that you run - do so by clicking on {% icon tool-versions %} Versions button.
+> To date, the newest version is **1.0.1** - the conda installation is much quicker in this version, but you may encounter some issues after switching kernel. Don't worry - if this happens, we will explain an alternative way of installation (takes a while though) in the relevant part of the tutorial.
+> If you use notebook version **1.0.0**, the conda installation is longer, but after it's done - the libraries should work fine even after switching kernel.
```

Comment:
I see ... the kernels are in different environments. So you need to change the kernel and then install the dependencies ...

Every kernel has its separated environment. Maybe that is important to explain?

Reply:
I added a note about that.

Somehow in the previous version of the notebook the libraries that were installed using conda at the beginning loaded fine after switching from bash to R - this works well for this particular tutorial however takes quite some time